docs/release_notes/0.4.0.md

```diff
@@ -0,0 +1,9 @@
+# Release 0.4.0
+
+## Features
+- ...
```

Comment:
Should we leave this line out until we have a feature to add here?

Reply:
We will need to review the notes before the release goes out anyway...

Reply:
```
## Features

Includes all features of the previous version!
```